packages/shared/src/components/modals/common.tsx

```diff
@@ -196,6 +200,7 @@ export const modals = {
   [LazyModal.ReportSource]: ReportSourceModal,
   [LazyModal.UserFollowersModal]: UserFollowersModal,
   [LazyModal.UserFollowingModal]: UserFollowingModal,
+  [LazyModal.Report]: ReportModal,
```

Comment:
Why is this called report? think it confuses things, I believe it's best to be called PostRejectModal

Reply:
This was just a reused pre-existing base modal we have. Maybe I can rename this to `ReasonSelectionModal` to be more clear.

Reply:
Would love to hear what you think here: https://github.com/dailydotdev/apps/pull/3656